UPDATING.md

```diff
@@ -48,94 +47,6 @@ assists users migrating to a new version.
 
 <!-- END doctoc generated TOC please keep comment here to allow auto update -->
 
-## Airflow 2.0.0a2
```

Comment:
Does deleting this records mean that we won't use `UPDATING.md` from 2.0 onward?

Reply:
I think that once 2.0 comes out we should create a new UPDATING.md and maybe move UPDATING to "UPDATING-1.0.md"